### ISSUE TITLE
Atlas downloading fails with capitalised names

### ISSUE THREAD

**Issue by adamltyson** [brainglobe/bg-atlasapi#132]
If the atlas name in `last_versions.conf` is capitalised, the atlas API seems to look for a version without the capitalisation (and fails). We can get around this by not capitalising atlas names (a good idea anyway for consistency), but it shouldn't cause an error.

**Comment by PolarBean** [comment]
can we close this? all atlas names are now lowercase

**Comment by adamltyson** [comment]
Should we add it to the validation?

**Comment by PolarBean** [comment]
good point. Shall I close this issue and create a new one for a lowercase validation function?

**Comment by adamltyson** [comment]
yep
